packages/@idemsInternational/gdrive-tools/src/commands/download.ts

```diff
@@ -95,6 +95,13 @@ export class GDriveDownloader {
     await this.setupGdrive();
     console.log(chalk.yellow("Retrieving list of files"));
     const serverFiles = await this.listGdriveFilesRecursively(folderId);
+    if (Object.keys(serverFiles).length === 0) {
+      console.log(
+        chalk.red("No files found. Failure to find files may result in an error.") +
+          "\n" +
+          chalk.red("Check if you are authorised to access the Google Drive folders.")
```

Comment:
I think this is a good place to log an error/warning.

I might phrase the message a bit differently:
"No asset files were found. Please ensure your Google account is authorised to access to relevant asset folder."